### ISSUE TITLE
Create family_tree table

### ISSUE THREAD

**Issue by owenbean400** [owenbean400/RecipeRootsSQL#6]
Create Family Tree table from ER diagram.

https://github.com/owenbean400/RecipeRoots/wiki/ER_Diagram

**Comment by owenbean400** [comment]
Waiting on pull request #20

**Comment by owenbean400** [comment]
This has been merged into main and can be closed
